src/test.js

```diff
@@ -719,15 +727,15 @@ function internalStart( test ) {
 
 function numberOfTests( module ) {
 	var count = module.tests.length;
-	while ( module = module.childModule ) {
+	while ( ( module = module.childModule ) ) {
```

Comment:
Forgot to add: [Here's the section](https://github.com/jquery/eslint-config-jquery/blob/master/.eslintrc.json#L4-L7) in eslint-config-jquery showing the "allow in parentheses" configuration I was describing. In other words, I didn't turn on the rule personally, but rather it came from the config we have chosen to use.

Again, if we want to change how it works in this repo, that's possible and totally okay.